.prettierrc

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
Why are these prettier rules being removed? Are they superseded by something else?

Reply:
the prettier rules were set to the default (check each of the defaults [here](https://prettier.io/docs/en/options.html)) so I don't think there's a point in keeping the file. Also it was kind of wrong because we were explicitly setting the parser to `"typescript"` when Prettier should just infer the parser based on the file type. Parsley doesn't have a `.prettierrc` file!